Makefile

```diff
@@ -0,0 +1,59 @@
+.PHONY: dummy_translations extract_translations fake_translations help pull_translations push_translations \
+
+
+.DEFAULT_GOAL := help
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## remove generated byte code, coverage reports, and build artifacts
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	coverage erase
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
+
+quality: ## check coding style with pycodestyle and pylint
+	echo 'TODO: Refactor quality checks from .travis.yml into this Makefile'
+	exit 1
+
+requirements: ## install development environment requirements
+	pip install -r requirements.txt --exists-action w
+	pip install -r requirements-dev.txt --exists-action w
+
+test: clean ## run tests in the current virtualenv
+	python run_tests.py
+
+selfcheck: ## check that the Makefile is well-formed
+	@echo "The Makefile is well-formed."
+
+## Localization targets
+
+extract_translations: ## extract strings to be translated, outputting .po files
+	rm -rf docs/_build
+	cd poll && django-admin makemessages -l en -v1 -d django
+	cd poll && django-admin makemessages -l en -v1 -d djangojs
+
+compile_translations: ## compile translation files, outputting .mo files for each supported language
+	django-admin compilemessages
+
+detect_changed_source_translations:
```

Comment:
The same comment as above goes here, but with the following error:
```
cd poll && i18n_tool changed
Traceback (most recent call last):
  File "/.virtualenvs/xblock-utils/bin/i18n_tool", line 11, in <module>
    sys.exit(main())
  File "/.virtualenvs/xblock-utils/lib/python2.7/site-packages/i18n/main.py", line 60, in main
    return module.main()
  File "/.virtualenvs/xblock-utils/lib/python2.7/site-packages/i18n/__init__.py", line 50, in __call__
    self.configuration = config.Configuration(filename=args.config, root_dir=root_dir)
  File "/.virtualenvs/xblock-utils/lib/python2.7/site-packages/i18n/config.py", line 37, in __init__
    self._config = self.read_config(self._filename)
  File "/.virtualenvs/xblock-utils/lib/python2.7/site-packages/i18n/config.py", line 62, in read_config
    raise Exception("Configuration file cannot be found: %s" % filename)
Exception: Configuration file cannot be found: /Projects/OpenCraft/xblock/xblock-poll/poll/conf/locale/config.yaml
make: *** [detect_changed_source_translations] Error 1
```

Reply:
Done